### PR TITLE
docs: update code sample of manually import echarts v5 modules

### LINKED ISSUE #406: Error when try to use modules imports
Everything works fine when I import the entire echarts library. But if I import modules I get the following error.
Снимок экрана 2021-02-10 в 22 25 58

code:
```
<ReactEchartsCore
        echarts={echarts}
        option={{
          xAxis: {
            type: 'category',
            data: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
          },
          yAxis: {
            type: 'value',
          },
          series: [
            {
              data: [150, 230, 224, 218, 135, 147, 260],
              type: 'line',
            },
          ],
        }}
        style={{ height: '200px' }}
        notMerge={true}
      />
```
`"echarts": "^5.0.2"`
`"echarts-for-react": "^3.0.0-beta.2"`

### PR DESCRIPTION
**Pull request by sjfkai**
Update code sample of manually import echarts v5 modules.
Fix #406

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,54 +64,76 @@ import ReactECharts from 'echarts-for-react';  // or var ReactECharts = require(
...
+  // VisualMapContinuousComponent,
+  // VisualMapPiecewiseComponent,
+  // AriaComponent,
+  // TransformComponent,
+  // DatasetComponent,
+} from 'echarts/components';
+// Import renderer, note that introducing the CanvasRenderer or SVGRenderer is a required step
+import {
+    CanvasRenderer,
+    // SVGRenderer,
```
Comment (hustcc) [inline README.md:130]: 2 spaces.

File: README.md [hunk 2]
```diff
@@ -64,54 +64,76 @@ import ReactECharts from 'echarts-for-react';  // or var ReactECharts = require(
...
+} from 'echarts/renderers';
+
+// Register the required components
+echarts.use(
+    [TitleComponent, TooltipComponent, GridComponent, BarChart, CanvasRenderer]
```
Comment (hustcc) [inline README.md:135]: 2 spaces.

**Review by hustcc — COMMENTED** [review]
Maybe we should distinguish between different versions and different usages.

v3 v4 has big difference with v5.

**Comment by hustcc** [comment]
you can add it into website with a live demo.

**Comment by sjfkai** [comment]
In README file, I added v5 code and did not remove v3 v4 code. Please check if there are any problems.

---

About live demo, I am not familiar with `dumi`. 
So there are some questions:
1. For  v5 demo, I can only import `ReactEChartsCore` by relative path
```ts
import ReactEChartsCore from '../../src/core'; // effective, but user can not use this
import ReactEChartsCore from 'echarts-for-react/lib/core'; // throw error, but user should use this
```
2. For v3 v4 demo, maybe we can not provide a live demo when the version of Echarts module is v5.

Any suggestions？

**Comment by hustcc** [comment]
Then don't add this demo.

merged.

**Pull request merged**